x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_about_rule_details/index.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiPanel,
+  EuiProgress,
+  EuiButtonGroup,
+  EuiButtonGroupOption,
+  EuiSpacer,
+  EuiFlexItem,
+  EuiText,
+  EuiFlexGroup,
+  EuiResizeObserver,
+} from '@elastic/eui';
+import React, { memo, useState } from 'react';
+import styled from 'styled-components';
+import { isEmpty } from 'lodash/fp';
+
+import { HeaderSection } from '../../../../../components/header_section';
+import { Markdown } from '../../../../../components/markdown';
+import { AboutStepRule, AboutStepRuleDetails } from '../../types';
+import * as i18n from './translations';
+import { StepAboutRule } from '../step_about_rule/';
+
+const MyPanel = styled(EuiPanel)`
+  position: relative;
+`;
+
+const FlexGroupFullHeight = styled(EuiFlexGroup)`
+  height: 100%;
+`;
+
+const VerticalOverflowContainer = styled.div((props: { maxHeight: number }) => ({
+  'max-height': `${props.maxHeight ?? '200'}px`,
```

Comment:
I don't think you need the `??` or you should add it the function below and set the type to accept a `number | null | undefined`

The usage is like so:
```
              <VerticalOverflowContainer
                data-test-subj="stepAboutDetailsNoteContent"
                maxHeight={aboutPanelHeight}
              >
                <VerticalOverflowContent
                  maxHeight={aboutPanelHeight}
                  className="eui-yScrollWithShadows"
                >
                  <Markdown raw={stepDataDetails.note} />
                </VerticalOverflowContent>
```

I think this is just a nit at this point, but pointing it out in case maybe I'm missing something.

Reply:
Updated! I don't in fact need the ??.